jscomp/others/belt_Array.ml

```diff
@@ -195,6 +195,15 @@ let slice a ~offset ~len =
       done ;
       result
 
+let sliceToEnd a ~offset  =
+  let lena = length a in
+  let ofs = if offset < 0 then max (lena + offset) 0 else offset in
```

Comment:
seems the label is not needed?
The general rule is that when there is no ambiguity involved, don't use label

Reply:
Right! I was so focused on the PR itself that I forgot about the actual usage/implem haha.
Fixed it.